tests/subaccounts/updateSubaccount.test.js

```diff
@@ -2,33 +2,20 @@ const testUtils = require('../testUtils.js')
 
 test('should update subaccount', async () => {
     let subaccount = await client.subaccounts.create('joske')
-    let email = testUtils.getRandomEmail()
-
-    await client.subaccounts.update(subaccount.id, 'jefke', email)
-
-    let retrievedSubaccount = await client.subaccounts.retrieve(subaccount.id)
-    expect(retrievedSubaccount.name).toBe('jefke')
-    expect(retrievedSubaccount.email).toBe(email)
-})
-
-test('email is optional', async () => {
-    let email = testUtils.getRandomEmail()
-    let subaccount = await client.subaccounts.createWithEmail(email, 'joske')
 
     await client.subaccounts.update(subaccount.id, 'jefke')
 
     let retrievedSubaccount = await client.subaccounts.retrieve(subaccount.id)
     expect(retrievedSubaccount.name).toBe('jefke')
-    expect(retrievedSubaccount.email).toBe(email)
 })
 
-test('name is not changed when updating with name=null', async () => {
+test('name is not changed when updating with name=null, email is ignored', async () => {
     let email = testUtils.getRandomEmail()
     let subaccount = await client.subaccounts.create('joske')
 
     await client.subaccounts.update(subaccount.id, null, email)
```

Comment:
Let's remove email as a parameter here; we don't want to imply to our customers that passing in the e-mail address still works.